test/cli/package-double-import-autocorrect/test.out

```diff
@@ -0,0 +1,52 @@
+same-file/b.rb:6: `A::Foo` resolves but its package is not imported https://srb.help/3718
+     6 |  sig {returns(A::Foo)}
+                       ^^^^^^
+    a/__package.rb:3: Exported from package here
+     3 |class A < PackageSpec
+        ^^^^^^^^^^^^^^^^^^^^^
+  Autocorrect: Done
+    same-file/__package.rb:3: Inserted `import A`
+     3 |class B < PackageSpec
+                             ^
+
+same-file/b.rb:8: `A::Foo` resolves but its package is not imported https://srb.help/3718
+     8 |    x = A::Foo.new
+                ^^^^^^
+    a/__package.rb:3: Exported from package here
+     3 |class A < PackageSpec
+        ^^^^^^^^^^^^^^^^^^^^^
+
+different-files/c2.rb:6: `A::Foo` resolves but its package is not imported https://srb.help/3718
+     6 |  sig{returns(A::Foo)}
+                      ^^^^^^
+    a/__package.rb:3: Exported from package here
+     3 |class A < PackageSpec
+        ^^^^^^^^^^^^^^^^^^^^^
+
+different-files/c1.rb:6: `A::Foo` resolves but its package is not imported https://srb.help/3718
+     6 |  sig{returns(A::Foo)}
+                      ^^^^^^
+    a/__package.rb:3: Exported from package here
+     3 |class A < PackageSpec
+        ^^^^^^^^^^^^^^^^^^^^^
+  Autocorrect: Done
+    different-files/__package.rb:3: Inserted `import A`
+     3 |class C < PackageSpec
+                             ^
+Errors: 4
+
+====================
+
+# typed: strict
+
+class B < PackageSpec
+  import A
```

Comment:
When I revert all the changes to `VisibilityChecker.cc` in this change, I still see only one `import` line added here, which suggests that this test does not actually exercise the behavior in this PR.

In particular, I'm guessing that the `hasSeen` logic in `AutcorrectSuggestion.cc` is preventing duplicate imports from being applied, but that would also mean that it's preventing non-duplicate imports from being added.

I think you should solve that problem first, then revisit this PR. Specifically:

- Given packages A1, A2, and B
- The constants A1::Foo and A2::Foo are defined inside (and exported from) their respective packages.
- Package B uses each of A1::Foo and A2::Foo twice.
- If you run `sorbet -a`, you only get one line added to the `__package.rb` file for B

Here's a longer test case.
[package-test.zip](https://github.com/sorbet/sorbet/files/14146482/package-test.zip)